app/models/user/UserService.scala

```diff
@@ -81,6 +82,12 @@ class UserService @Inject()(conf: WkConf,
       _ <- bool2Fox(userBox.isEmpty) ?~> "organization.alreadyJoined"
     } yield ()
 
+  def assertIsSuperUser(multiUserId: ObjectId)(implicit ctx: DBAccessContext): Fox[Unit] =
+    for {
+      isSuperUser <- multiUserDAO.findOne(multiUserId).map(_.isSuperUser)
+      _ <- bool2Fox(isSuperUser)
+    } yield ()
```

Comment:
```suggestion
Fox.assertTrue(multiUserDAO.findOne(multiUserId).map(_.isSuperUser))
```
(could you try that? I think that’s the same but maybe I’m overlooking something)